src/Compilers/CSharp/Portable/Lowering/SyntheticBoundNodeFactory.cs

```diff
@@ -1304,6 +1349,12 @@ public BoundExpression InstrumentationPayloadRoot(int analysisKind, TypeSymbol p
             return new BoundInstrumentationPayloadRoot(Syntax, analysisKind, payloadType) { WasCompilerGenerated = true };
         }
 
+        public BoundExpression ThrowIfModuleCancellationRequested()
+            => new BoundThrowIfModuleCancellationRequested(Syntax, SpecialType(CodeAnalysis.SpecialType.System_Void)) { WasCompilerGenerated = true };
```

Comment:
>new BoundThrowIfModuleCancellationRequested(Syntax, SpecialType(CodeAnalysis.SpecialType.System_Void)) { WasCompilerGenerated = true  [](http://example.com/codeflow?start=15&length=133)

It doesn't look like we are making sure the ```System_Threading_CancellationToken``` type and the throw method are available.

Reply:
We are here - https://github.com/dotnet/roslyn/pull/71896/files#diff-61e54648a0c8e35adbcd040f5e0bff216826c00c393396aaa72e418682be5cbdR48-R52